fbfmaproom/fbflayout.py

```diff
@@ -403,87 +403,28 @@ def table_layout():
             dcc.Loading(
                 [
                     table.DataTable(
-                        id="summary",
+                        id="table",
                         page_action="none",
                         style_table={
-                            "height": "auto",
-                            "overflowY": "scroll",
```

Comment:
The overflowY property ends up being unnecessary (whether because of fixed rows or just period) and if added introduces undesirable visual artifacts

Reply:
Take a look at dash-spreadsheet dash-freeze-top